test/staking.ts

```diff
@@ -92,14 +93,35 @@ describe("Staking contract", function () {
   });
 
   describe("Unstake", () => {
+    const numInitialValidators = 5;
     const stakedAmount = ethers.utils.parseEther("1");
+    const stake = async (account: SignerWithAddress, value: BigNumber) => {
+      return contract.connect(account).stake({ value });
+    };
     beforeEach(async () => {
-      await contract.stake({ value: stakedAmount });
+      // set required number of validators
+      await Promise.all(
+        accounts
+          .slice(0, numInitialValidators)
+          .map((account) => stake(account, stakedAmount))
+      );
     });
 
     it("should failed if an account is not a staker", async () => {
-      await expect(contract.connect(accounts[1]).unstake()).to.be.revertedWith(
-        "Only staker can call function"
+      await expect(
+        contract.connect(accounts[numInitialValidators]).unstake()
+      ).to.be.revertedWith("Only staker can call function");
+    });
+
+    it("should failed if current number of validators equals to MinimumRequiredNumValidators", async () => {
+      // remove 1 account first
+      await contract.connect(accounts[1]).unstake();
```

Comment:
I think there is no need to remove this account here.

Reply:
I'm afraid I don't think so, the purpose of this test is making sure any validator can't unstake if the number of validator equals to `MinimumRequiredNumValidators`. At the beginning of the test, the number is 5 while MinimumRequiredNumValidators is 4. We need to decrease the number first.